dependencies.gradle

```diff
@@ -1,48 +1,26 @@
 // Add your dependencies here
 
 dependencies {
-    compile("com.github.GTNewHorizons:GT5-Unofficial:5.09.40.32:dev")
-    compile("com.github.GTNewHorizons:StructureLib:1.0.15:dev") 
-    compile("com.github.GTNewHorizons:NotEnoughItems:2.2.6-GTNH:dev") 
-    compile("com.github.GTNewHorizons:CodeChickenCore:1.1.3:dev")
-    compile("com.github.GTNewHorizons:CodeChickenLib:1.1.5.3:dev")
-    compile("net.industrial-craft:industrialcraft-2:2.2.828-experimental:dev")
+    compile('com.github.GTNewHorizons:GT5-Unofficial:5.09.40.35:dev')
```

Comment:
Why are deps bumped every time anyone does anything? I thought the point of explicit deps was so that didn’t happen unless the updated deps were actually required…

Reply:
Things are still in flux and transitive deps are being added.  Updating or not updating, as long as the builds are reproducible, and the underlying API hasn't changed,  shouldn't matter in practice.

Reply:
Once the whole thing calms down we'll be able to leave them as is; but right now too many things are being changed (mainly the build scripts :( )